servers/rendering/renderer_rd/storage_rd/material_storage.h

```diff
@@ -100,8 +100,8 @@ class MaterialStorage : public RendererMaterialStorage {
 		HashMap<StringName, uint64_t> used_global_textures;
 
 		//internally by update_parameters_uniform_set
-		Vector<uint8_t> ubo_data;
-		RID uniform_buffer;
+		Vector<uint8_t> ubo_data[2]; // 0: linear buffer; 1: sRGB buffer.
```

Comment:
I would use an enum for the buffer, its just clearer why we have two. Might be over kill.

Reply:
Could you clarify where the Enum would go?

Do you mean you would create an enum just to establish the size of the Array?

Reply:
I believe what he means is making it clearer that 0 is Linear and 1 is sRGB through an Enum. Declared at the top of material.h like this:

```
enum BufferType {
	LINEAR = 0,
	SRGB = 1
};
```

Used as such (without the need of comments):
```
Vector<uint8_t> ubo_data[BufferType.LINEAR];
RID uniform_buffer[BufferType.SRGB];
```

I don't find it overkill, might save some headache for someone trying to guess which one is which in a later feature. But I think this ship has sailed since it has already been merged.

@clayjohn if this is welcome, let me know if I should make another PR with this minor change. It might be a good start for me to get used with contributions.

Reply:
I don't think it would be beneficial. Using enums hides what is actually happening. For example, your code will fail to compile (an array of size 0 doesn't work for the UBO array, the array needs to have 2 elements). Also, the uniform buffer RID array is also wrong since it also needs two elements. 

It's really easy to make this kind of mistake when you hide the important values behind an enum. 

That being said, enums can be really handy when you use them both to initialize the array and to index into the array. Doing something like:
```
LINEAR = 0 // use to index into the array
SRGB = 1 // use to index into the array
MAX = 2 // use to initialize the array
```

But in this case we index into the array with a boolean. So enums won't help. 

IMO using enums here adds more complexity for no gain